### PR TITLE
Irq fix

### PR DESCRIPTION
**Pull request by gmartin102**
This updates fc_subsystem to correct the interrupt inputs to the cv32e40p.  they are connected to the unmasked irq output array from the apb_interrupt_cntrl

### CODE REVIEW

**Comment by gmartin102** [comment]
I attempted to resolve the conflicts.. mostly formatting issues, except for ioscript.py

File: util/ioscript.py [hunk 1]
```diff
@@ -870,11 +844,11 @@ def evaluate_defines(x):
                     x_sv.write("    .I(xilinx_io[%d]),\n" % sysionames.index("jtag_tck_o"))
                     x_sv.write("    .O(s_io[%d])\n" % sysionames.index("jtag_tck_o"))
                     x_sv.write("  );\n\n")
-
+ 
         # print remaining connections, if any
         if ionum_end != -1:
             x_sv.write("  assign s_io[%d:%d] = xilinx_io[%d:%d];\n\n" % (ionum_end, ionum_start, ionum_end, ionum_start))
-
+
```
Comment (zarubaf) [inline util/ioscript.py:851]: There seems to be trailing whitespaces. Can you set your editor to trim those, please?

File: util/ioscript.py [hunk 2]
```diff
@@ -870,11 +844,11 @@ def evaluate_defines(x):
                     x_sv.write("    .I(xilinx_io[%d]),\n" % sysionames.index("jtag_tck_o"))
                     x_sv.write("    .O(s_io[%d])\n" % sysionames.index("jtag_tck_o"))
                     x_sv.write("  );\n\n")
-
+
```
Comment (zarubaf) [inline util/ioscript.py:847]: Trailing space.

**Review by zarubaf — CHANGES_REQUESTED** [review]

**Comment by zarubaf** [comment]
I though I wrote you am answer here but I can’t seem to find it. Just rerun the ˋmake allˋ in the docs folder and commit the docs, then we are good to go!

**Comment by zarubaf** [comment]
Okay Tests are passing. Just briefly want to check with you that you are okay with the updated doc (you can view them in your repo or see the new doc in the diff) then I’ll merge. To me it looks the IO script broke because of a merge. Let me know!

**Comment by gmartin102** [comment]
Closed and opeing a new pull request against master

**Pull request closed without merging**